marketplaceItems/drawApp/fingerDraw_app.js

```diff
@@ -0,0 +1,596 @@
+//
+//  draw_app.js
+//
+//  Created by Rebecca Stankus on 01/31/19
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+/* global  Audio, Camera, Controller, Entities, HMD, Messages, MyAvatar, Quat, Script, Settings, SoundCache, 
+Tablet, Vec3, Window */
+
+(function() {
+    var REPEAT_DISTANCE_CHECK_MS = 60;
```

Comment:
please move these vars and consts closer to where they're first used, or closer to where they're most relevant